test/unit/toggle/toggle-component.js

```diff
@@ -0,0 +1,147 @@
+import Toggle from '../../../src/components/toggle';
+import ToggleView from '../../../src/views/toggle';
+
+describe('Toggle Component class', () => {
+  let toggle;
+  let cart;
+  let node;
+
+  beforeEach(() => {
+    node = document.createElement('div');
+  });
+
+  describe('constructor', () => {
+    beforeEach(() => {
+      cart = {
+        node: document.createElement('div'),
```

Comment:
Since we're passing in `node` for all tests but one, we can probably move the contents of the beforeEach and afterEach into that test